lib/tilex_web/controllers/channel_controller.ex

```diff
@@ -17,7 +17,27 @@ defmodule TilexWeb.ChannelController do
       posts: posts,
       posts_count: posts_count,
       channel: channel,
-      page: page
+      page: page,
+      random: false
+    )
+  end
+
+  def random_by_channel(conn, %{"channel" => channel_name} = params) do
+    page =
+      params
+      |> Map.get("page", "1")
+      |> String.to_integer()
```

Comment:
Think this variable is unused?

Reply:
My bad, overlooked that. Meant to remove it. Took care of it!